public/index.html

```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
   <div class="content-wrapper">
     
     <div class="subtitle has-text-centered">
-      <h4>Implementation</h4>
+      <h4>Implementations</h4>
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/rudr.svg" alt="Rudr" /></div>
+      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
       <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
-        <small><a href="https://github.com/oam-dev/rudr">github.com/oam-dev/rudr</a></small>
+        <h1>Crossplane: A Kubernetes implementation of the Open Application Model</h1>
+        <small><a href="https://crossplane.io">crossplane.io</a></small>
       </div>
       
     </div>
 
     <div class="columns">
       <div class="column is-quarter has-text-left">
         <ul>
-          <li><a href="https://github.com/oam-dev/rudr/tree/master/docs">Documentation</a></li>
-          <li><a href="https://github.com/oam-dev/rudr/blob/master/docs/setup/install.md">Getting Started</a></li>
-          <li><a href="https://github.com/oam-dev/rudr">Rudr Github Repo</a></li>
+          <li><a href="https://crossplane.io/docs/latest">Documentation</a></li>
+          <li><a href="https://github.com/crossplane/crossplane/blob/master/docs/getting-started/install.md">Getting Started</a></li>
```

Comment:
```suggestion
          <li><a href="https://crossplane.io/docs/latest">Getting Started</a></li>
```
Docs are changing so best to link to the docs landing page.